woocommerce/woocommerce-ios
# [Mobile Payments] Remove direct access to cell views

Resolves #3771 

Changes:
- Added `configure` methods to the table cell views accepting either a single value, or a viewmodel for mutliple values - to bring the card reader table cell views we added to harmony with how other table cell views were written
- Removed `TitleTableViewCell` entirely as it wasn't being used anywhere
- Added `UIImage.cardReaderConnect` and a test for it to simplify setting up ImageTableViewCells

To test:
- No visual changes should result from these changes. Connect and disconnect a card reader and ensure all text and images are as expected. Ensure the "learn more" link continues to open the URL as before.
- Ensure the newly added "icon" test for cardReaderConnect image passes

Update release notes:
